src/commissaire_http/handlers/clusters.py

```diff
@@ -224,11 +230,15 @@ def add_cluster_member(message, bus):
                 message['id'],
                 [message['params']['host']]
             )
+        else:
```

Comment:
This `else:` looks incorrect to me.  You want to return a response either way, right?